src/amo/components/Link.js

```diff
@@ -18,7 +19,12 @@ export class LinkBase extends React.Component {
   }
 
   render() {
-    const { base, children, to } = this.props;
+    const { base, children, href, to } = this.props;
```

Comment:
Ah, fair enough. The `Link` component doesn't do that but no reason we can't be stricter 👍